src/main/java/com/dataguise/saas/dto/SchedulerDTO.java

```diff
@@ -0,0 +1,170 @@
+
+    private Integer id = null;
+
+    private String moduleType = null;
+
+    @JsonIgnore
+    public List<TaskParamsDTO> getTaskParams() {
+        return taskParams;
+    }
+
+    @JsonProperty
+    public void setTaskParams(List<TaskParamsDTO> taskParams) {
+        this.taskParams = taskParams;
+    }
+
+    @JsonIgnore
+    public String getScheduleName() {
+        return scheduleName;
+    }
+
+    @JsonProperty
+    public void setScheduleName(String scheduleName) {
+        this.scheduleName = scheduleName;
+    }
+
+    @JsonIgnore
+    public String getScheduleType() {
+        return scheduleType;
+    }
+
+    @JsonProperty
+    public void setScheduleType(String scheduleType) {
+        this.scheduleType = scheduleType;
+    }
+
+    @JsonIgnore
+    public String getScheduleInterval() {
+        return scheduleInterval;
+    }
+
+    @JsonProperty
+    public void setScheduleInterval(String scheduleInterval) {
+        this.scheduleInterval = scheduleInterval;
+    }
+
+    @JsonIgnore
+    public Integer getId() {
+        return id;
+    }
+
+    @JsonProperty
+    public void setId(Integer id) {
+        this.id = id;
+    }
+
+    @JsonIgnore
+    public String getModuleType() {
+        return moduleType;
+    }
+
+    @JsonProperty
+    public void setModuleType(String moduleType) {
+        this.moduleType = moduleType;
+    }
+
+    @JsonIgnore
+    public List<String> getScheduleDay() {
+        return scheduleDay;
+    }
+
+    @JsonProperty
+    public void setScheduleDay(List<String> scheduleDay) {
+        this.scheduleDay = scheduleDay;
+    }
+
+    @JsonIgnore
+    public String getScheduleScanEndDate() {
+        return scheduleScanEndDate;
+    }
+
+    @JsonProperty
+    public void setScheduleScanEndDate(String scheduleScanEndDate) {
+        this.scheduleScanEndDate = scheduleScanEndDate;
+    }
+
+    @JsonIgnore
+    public String getStartDateTime() {
+        return startDateTime;
+    }
+
+    @JsonProperty
+    public void setStartDateTime(String startDateTime) {
+        this.startDateTime = startDateTime;
+    }
+
+    @JsonIgnore
+    public String getScanType() {
+        return scanType;
+    }
+
+    @JsonProperty
+    public void setScanType(String scanType) {
+        this.scanType = scanType;
+    }
+
+    public DgTaskScheduleInfoBean getSchedule() {
+        return schedule;
+    }
+
+    public void setSchedule(DgTaskScheduleInfoBean schedule) {
+        this.schedule = schedule;
+    }
+
+    public DgTaskScheduledExecutionsBean getScheduleSummary() {
+        return scheduleSummary;
+    }
+
+    public void setScheduleSummary(DgTaskScheduledExecutionsBean scheduleSummary) {
+        this.scheduleSummary = scheduleSummary;
+    }
+    public String getSourceModule() {
+        return sourceModule;
+    }
+
+    public List<String> getScanLocations() { return scanLocations; }
+
+    public void setScanLocations(List<String> scanLocations) { this.scanLocations = scanLocations; }
```

Comment:
**Reporter**: Checkstyle    
    **Rule**: com.puppycrawl.tools.checkstyle.checks.sizes.LineLengthCheck
    **Severity**: ERROR
    **File**: /work/src/main/java/com/dataguise/saas/dto/SchedulerDTO.java L169
    
    Line is longer than 80 characters (found 100).
    
*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a273302203>*